apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx

```diff
@@ -19,7 +19,7 @@ const LanguageInformationSection = ({
   const intl = useIntl();
   return (
     <Well>
-      {!hasAllEmptyFields(user) && <Display user={user} context="admin" />}
+      {!hasAllEmptyFields(user) && <Display user={user} />}
```

Comment:
[Line 24](https://github.com/GCTC-NTGC/gc-digital-talent/blob/98bdb8a813c4338c73e96a4d7c12c7c5ced18dda/apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx#L24) needs to be migrated to tailwind